src/plugins/translation-ui-router.js

```diff
@@ -0,0 +1,80 @@
+const _ = require('lodash');
+const { createFilter } = require('rollup-pluginutils');
+const { walk } = require('estree-walker');
+const MagicString = require('magic-string');
+const utils = require('./translation-utils');
+
+const removeProperty = (code, magicString, start, end) => {
+  magicString.remove(start, end);
+  for (let i = end; i < code.length - 1; i += 1) {
+    const ch = code[i];
+    if (ch === ',') {
+      magicString.remove(i, i + 1);
+      break;
+    } else if (/\w|}/.test(ch)) {
+      break;
+    }
+  }
+};
+
+module.exports = (opts = {}) => {
+  const include = opts.include || '**/*.js';
+  const { exclude } = opts;
+  const filter = createFilter(include, exclude);
+  const sourcemap = opts.sourcemap !== false;
+  const subdirectory = opts.subdirectory || './';
+  return {
+    name: 'translation-ui-router',
+    transform(code, id) {
+      if (!filter(id)) return null;
+      const ast = this.parse(code);
+      const magicString = new MagicString(code);
+      walk(ast, {
+        enter(node) {
+          if (sourcemap) {
+            magicString.addSourcemapLocation(node.start);
+            magicString.addSourcemapLocation(node.end);
+          }
+          if (_.get(node, 'callee.object.name') === '$stateProvider') {
+            const props = _.chain(node)
+              .get('arguments')
+              .last()
+              .get('properties')
+              .value();
+            const trads = _.chain(props)
+              .filter({ key: { name: 'translations' } })
+              .filter({ type: 'Property' })
+              .head();
+
+            if (trads.value()) {
+              const resolve = _.chain(props)
+                .filter({ key: { name: 'resolve' } })
```

Comment:
Why don't merge theses 2 filters ? 

```suggestion
                .filter({ key: { name: 'resolve' }, type: 'Property' })
```